src/lib/merkle_mask/masking_merkle_tree_intf.ml

```diff
@@ -15,6 +15,8 @@ module type S = sig
 
   type key
 
+  module Location : Merkle_ledger.Location_intf.S
+
   module Addr : Merkle_address.S
```

Comment:
Can you talk make this signature include base merkle ledger as well and extend from there as well?

Reply:
There's an `include` for the submodule `Attached` already. The idea is, you don't want to perform any operation for the unattached mask. 

We could add those operations for the unattached mask, but they all should fail in that case.